packages/fxa-payments-server/src/routes/Subscriptions/index.tsx

```diff
@@ -323,36 +317,27 @@ export const Subscriptions = ({
 
 const customerSubscriptionForId = (
   subscriptionId: string,
-  customerSubscriptions: SubscriptionsProps['customerSubscriptions']
+  customerSubscriptions: CustomerSubscription[]
 ): CustomerSubscription | null =>
-  !customerSubscriptions
-    ? null
-    : customerSubscriptions.filter(
-        subscription => subscription.subscription_id === subscriptionId
-      )[0];
+  customerSubscriptions.filter(
+    subscription => subscription.subscription_id === subscriptionId
+  )[0];
 
 const subscriptionForId = (
   subscriptionId: string,
-  subscriptions: SubscriptionsProps['subscriptions']
+  subscriptions: Subscription[]
 ): Subscription | null =>
-  !subscriptions.result
-    ? null
-    : subscriptions.result.filter(
-        subscription => subscription.subscriptionId === subscriptionId
-      )[0];
-
-const planForId = (
-  planId: string,
-  plans: SubscriptionsProps['plans']
-): Plan | null =>
-  !plans.result
```

Comment:
Don't need to guard against null `plans.result` here, because it's happening earlier with error dialogs.